### DISCUSSION TITLE
redirect url to ip

### DISCUSSION THREAD

**Discussion by Voo2** [AdguardTeam/AdGuardHome#6953] [Q&A]
I actually want to do something very simple, but I can't manage it. I would like someone accessing the address test.myserver.de to be redirected to the IP 192.168.178.39 in the local network.

I have created a custom filter for this purpose:

192.168.178.39 test.myserver.de

I have also entered the new DNS server in my network card settings.

However, this does not work when I try it in my browser or send a ping. What am I doing wrong?

**Comment by ghost** [comment]
`||example.com^$dnsrewrite=1.2.3.4`

https://adguard-dns.io/kb/general/dns-filtering-syntax/#dnsrewrite-modifier

**Comment by Voo2** [comment]
Thank you for your answer. Correct me If I'm wrong, but I tried to open a shell and ping test.myserver.de. This still doesn't work. I got the message that this host couldn't be found.
